macos_build.sh

```diff
@@ -8,8 +8,8 @@ git -C vcpkg apply ../vcpkg_fixup_pkgconfig.cmake.diff
 vcpkg/bootstrap-vcpkg.sh
 
 TRIPLET="--overlay-triplets=. --triplet=x64-osx-openrct2"
-LIBRARIES="duktape freetype[core,zlib,bzip2,png] libpng libzip[core] nlohmann-json openssl sdl2 speexdsp discord-rpc"
-vcpkg/vcpkg install ${=TRIPLET} ${=LIBRARIES}  
+LIBRARIES="duktape freetype[core,zlib,bzip2,png] libpng libzip[core] nlohmann-json openssl sdl2 speexdsp discord-rpc icu"
```

Comment:
While we're here, I set the freetype build to exclude the `brotli` feature simply to avoid freetype from triggering any extra library builds. However, since we're adding icu, zlib, and bzip2 here, there's not much reason to *not* include libbrotli, especially if it has a functional use (enabling WOFF2 fonts). This can be done by removing the entire `[...]` part for freetype and having it build all the default features.